src/e2e/app/cli/common/src/kamu_api_server_client.rs

```diff
@@ -7,15 +7,20 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+use chrono::{DateTime, Utc};
 use internal_error::{InternalError, ResultIntoInternal};
-use opendatafabric::DatasetAlias;
 use reqwest::{Method, Response, StatusCode, Url};
 use serde::Deserialize;
+use serde_json::json;
 use tokio_retry::strategy::FixedInterval;
 use tokio_retry::Retry;
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+pub type AccessToken = String;
```

Comment:
This is a bit confusing, as we have such type in the application.
Also, maybe we should rather use a tuple struct?